modules/dxp/apps/osb/osb-spring-boot/osb-spring-boot-client-zendesk/src/main/java/com/liferay/osb/spring/boot/client/zendesk/service/ZendeskService.java

```diff
@@ -294,6 +295,23 @@ public void updateZendeskTicket(
 		).block();
 	}
 
+	public void updateZendeskTickets(JSONArray jsonArray) throws Exception {
```

Comment:
can this take List<ZendeskTicket> instead?

Reply:
jsonarray is unstructured so it's harder to tell how to pass it in.
This also keeps the transform logic when sending to zendesk in one place.